### ISSUE TITLE
Dev: fix Alert: Missing fieldset on Security page

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/tdm-calculator#3036]
### Overview
We need to fix the Alert(s): Missing fieldset on Security so that page can be WCAG compliant.

#### Details
Category: Alert
Name: Missing fieldset
(The Accessibility wiki for this Alert has no solution yet)

Page that will be worked on:
 - Security

### Action Items
- [ ] Review wiki page: Accessibility, How to run the WAVE browser extension (Resource 1.01)
   - [ ] Install the WAVE browser extension
- [ ] Log in as the user indicated (resource 1.02.01) 
- [ ] navigate to the page you are going to fix (resource 1.02.02)
- [ ] Open the TDM WAVE Accessibility Audit Spreadsheet, for this Alert (resource 1.03) and find the row that relates to this  issues page.
- [ ] Run the WAVE extension with no elements focused, or clicked (resource 1.04)
- [ ] Find the Alert shown in the Spreadsheet (1.03)
- [ ] Create and apply solution
- [ ] Write and add the appropriate details to the `Project Team Solution` section (resource 1.05.01.01)
- [ ] Submit PR (include link to `Project Team Solution` section (resource 1.05.01.01)
- [ ] Add PR to spreadsheet in PR column next to Alert (resource 1.03)
- [ ] On this issue
   - [ ] Add labels: 
      - `ready for accessibility lead`
      - `ready for product`
   - [ ] Change status to: `Technical Review`
- [ ] Get sign-off on WIKI page by
   - [ ] Accessibility lead
   - [ ] Product lead
- [ ] release the dependenciy on #2894

### Resources/Instructions
- 1.01 [How to run the WAVE browser extension](https://hackforla.github.io/accessibility/getting-started/wave-browser-extension)
- 1.02 Application details
   - 1.02.01 User Profile Required: security admin
   - 1.02.02 Page URL: https://tdm-dev.azurewebsites.net/roles
      - Requires sign in: TRUE
      - Can access page directly from URL: FALSE
- 1.03 [TDM WAVE Accessibility Audit Spreadsheet, tab: 2. Audit Log, filter: Alert: Missing fieldset](https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=0#gid=0&fvid=1221784535)
   -  Spreadsheet Row(s): 4, 5 & 6
- 1.04 https://hackforla.github.io/accessibility/getting-started/perform-wave-audit
- 1.05 [Accessibility Known Issues Index](https://hackforla.github.io/accessibility/known-issues-and-solutions/solutions-index)
    - 1.05.01 [WAVE Known Accessibility Issue: Alert - Missing fieldset](https://hackforla.github.io/accessibility/known-issues-and-solutions/solutions-index/alert-missing-fieldset)
       - 1.05.01.01 [WAVE Known Accessibility Issue: Alert - Missing fieldset, Project Team Solution](https://hackforla.github.io/accessibility/known-issues-and-solutions/solutions-index/alert-missing-fieldset/#project-team-solution)

Additional reading (optional)
- TDM [Accessibility Audits](https://hackforla.github.io/accessibility/)

#### Slides
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg)
   - [Staging Deck, slide 3036](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g39673b86911_4_0#slide=id.g39673b86911_4_0)
- Release deck
   - [2026-MM-DD Release deck, slide 3036]

**Comment by Rabia2219** [comment]
@geolunalg Bonnie and I removed the `size: 1pt` label because this issue requires the developer to create a solution, apply it to the issue and write the solution into the WIKI page on the accessibility repo.

Please add a level label. You can choose from one of the following:

- ~`good first issue`~
- `level: easy`
- `level: medium`
- `level: hard`

Please add a size label. You can choose from one of the following:

| label        | Description                             |
| ------------ | --------------------------------------- |
| ~size: 0.25pt~ |~Can be done in 1.5 hours or less~        |
| ~size: 0.5pt~  | ~Can be done in 2-3 hours or less~        |
| ~size: 1pt~    | ~Can be done in 4-6 hours~              |
| size: 2pt    | Can be done in 7-12 hours               |
| size: 3pt    | Can be done in 13-18 hours              |
| size: 5pt    | Can be done in 19-30 hours              |
| size: 8pt    | Can be done in 31-48 hours              |
| size: 13+pt  | Must be broken down into smaller issues |


Once you have added the level and size labels, please:
- remove the `level: missing` label
- remove the `size: missing` label
- remove the `ready for accessibility lead` label
- add the `ready for prioritization` label
